great_expectations/rule_based_profiler/parameter_builder/mean_table_columns_set_match_multi_batch_parameter_builder.py

```diff
@@ -0,0 +1,142 @@
+from typing import Dict, List, Optional, Set, Union
+
+import numpy as np
+
+from great_expectations.rule_based_profiler.config import ParameterBuilderConfig
+from great_expectations.rule_based_profiler.helpers.util import (
+    get_parameter_value_and_validate_return_type,
+)
+from great_expectations.rule_based_profiler.parameter_builder import (
+    MetricMultiBatchParameterBuilder,
+)
+from great_expectations.rule_based_profiler.types import (
+    FULLY_QUALIFIED_PARAMETER_NAME_METADATA_KEY,
+    FULLY_QUALIFIED_PARAMETER_NAME_VALUE_KEY,
+    Domain,
+    MetricValue,
+    MetricValues,
+    ParameterContainer,
+    ParameterNode,
+)
+from great_expectations.types.attributes import Attributes
+
+
+class MeanTableColumnsSetMatchMultiBatchParameterBuilder(
+    MetricMultiBatchParameterBuilder
+):
+    """
+    Compute mean match ratio (as a fraction) of "table.columns" metric across every Batch of data given.
```

Comment:
Could we add the description in the PR to this Doc string? I think it would help a new user/reviewer get the necessary context in this ParameterBuilder.